low-level/sensors/impl/centos_7/disk_monitor.py

```diff
@@ -216,10 +221,20 @@ def run(self):
         try:
             # Connect to dbus system wide
             self._bus = SystemBus()
+            try:
+                # Obtain a disk manager interface for monitoring drives
+                disk_systemd = self._bus.get_object('org.freedesktop.UDisks2',
+                    '/org/freedesktop/UDisks2')
+                self._iem.iem_fault_resolved("UDISKS2_UNAVAILABLE", "UDISKS2_AVAILABLE")
+            except dbus.DBusException as err:
+                logger.error("DiskMonitor: Error occurred while"
+                    " initializing dbus UDisks interface. : %s" %err)
+                if self.UDISKS2_UNAVAILABLE in str(err):
+                    self._iem.iem_fault("UDISKS2_UNAVAILABLE")
+                    return

```

Comment:
if in case udisks debus interface is not loaded we are logging IEM and  if we proceed without return from here sensor thread will encounter an errors and thread will be shutdown.